### DISCUSSION TITLE
Where can I initiate the ChatGPT prompt for Full Web Page Content?

### DISCUSSION THREAD

**Discussion by kdalex** [plateaukao/einkbro#376] [Q&A] [answered]
I searched multiple times for the right menu item to initiate the "Prompt for Full Web Page Content" in Einkbro. I'm running 11.10.0 and I have saved my API key in the settings.

I can initiate a prompt based on selected text, but how do use the feature for the whole page? The prompt for that is stored in the settings too ("Summarise in 300 words:")

**Comment by plateaukao** [comment, marked as answer]
it's....very hidden. You have to add the "page count" button to the toolbar; and when you click on it, it will trigger chatgpt acton for whole web page.
